javascript-timers/interval.js

```diff
@@ -0,0 +1,14 @@
+let count = 4;
+
+var counter = setInterval(timer, 1000); // 1000 will  run it every 1 second
+var theText = document.querySelector('.countdown-display');
+
+function timer() {
+  count = count - 1;
+  if (count <= 0) {
+    clearInterval(counter);
+    theText.innerHTML = '~Earth Beeeeelow Us~';
+  } else {
+    theText.innerHTML = count;
```

Comment:
Setting `innerHTML` is generally a security hole. It's better to use DOM creation or to set `textContent`.